src/cpp/core/include/core/spelling/HunspellDictionaryManager.hpp

```diff
@@ -93,7 +93,9 @@ class HunspellDictionaryManager
 
 private:
    core::FilePath allLanguagesDir() const;
-   core::FilePath userLanguagesDir() const;
+   core::FilePath customLanguagesDir() const;
+   core::FilePath old_allLanguagesDir() const;
```

Comment:
We don't usually mix snake/camel case (recommend `oldAllLanugagesDir` or `legacyAllLanguagesDir`)